apps/public/src/app/(map)/locatie/kies/page.test.tsx

```diff
@@ -1,15 +1,23 @@
-import { render, waitFor } from '@testing-library/react'
+import { render, screen, waitFor } from '@testing-library/react'
 
 import Page from './page'
 
 describe('Page', () => {
-  it('should render', async () => {
+  it('should render', () => {
     const { container } = render(<Page />)
 
     const outerWrapper = container.querySelector('div')
 
     waitFor(() => {
       expect(outerWrapper).toBeInTheDocument()
     })
+
+    const sideBar = screen.getByRole('heading', { name: 'Selecteer de locatie' })
+    const addressComboBox = screen.getByRole('combobox', { name: 'Zoek op adres' })
+    const toggleButton = screen.getByRole('button', { name: 'Lijst' })
+
+    expect(sideBar).toBeInTheDocument()
+    expect(addressComboBox).toBeInTheDocument()
+    expect(toggleButton).toBeInTheDocument()
```

Comment:
We're not really testing the toggling behaviour atm, I think? Should we add that?